deps/csl.mk

```diff
@@ -125,8 +125,22 @@ install-csl:
 	cp -a $(build_libdir)/gcc/$(BB_TRIPLET)/$(GCC_VERSION)/libmsvcrt.a $(build_private_libdir)/
 	cp -a $(build_libdir)/gcc/$(BB_TRIPLET)/$(GCC_VERSION)/libssp.dll.a $(build_private_libdir)/
 	cp -a $(build_libdir)/gcc/$(BB_TRIPLET)/$(GCC_VERSION)/libssp.dll.a $(build_libdir)/
+ifeq ($(ARCH),x86_64)
+#	libmingw32:		mingw_raise_matherr
+#	libmingwex:		ldexp,log2l,exp2l,powi
+	-cp -a /mingw64/lib/libmsvcrt.a     $(build_private_libdir)/
+	-cp -a /mingw64/lib/libmingw32.a    $(build_private_libdir)/
+	-cp -a /mingw64/lib/libmingwex.a    $(build_private_libdir)/
```

Comment:
Maybe we want to add `libmingw32`, `libmingwex` to CSL too.

And if I use `libmsvcrt.a` from CSL, it failed to build.
Seems there is some version mismatch.